xla_native_functions.yaml

```diff
@@ -118,13 +122,6 @@ supported:
   - bernoulli_.float
   - bernoulli_.Tensor
   - binary_cross_entropy_with_logits
-  - bitwise_and.Scalar
```

Comment:
any reason we remove all scalar version of overwrites? Did you verify that when user call `torch.bitwise_and` for `tensor, scalar` it got dispatched to tensor version?

Reply:
Yes, testing with the following shows that `xla::bitwise_and` gets executed on scalar operations.
```
>>> import torch; import torch_xla
>>> torch.bitwise_and(torch.tensor(100,device='xla:0'), 10)
tensor(0, device='xla:0')
>>> import torch_xla.debug.metrics as met; print(met.metrics_report())
```